script/RumpelConfig.sol

```diff
@@ -40,6 +40,11 @@ library RumpelConfig {
     address public constant MAINNET_ZIRCUIT_RESTAKING_POOL = 0xF047ab4c75cebf0eB9ed34Ae2c186f3611aEAfa6;
     address public constant MAINNET_SYMBIOTIC_WSTETH_COLLATERAL = 0xC329400492c6ff2438472D4651Ad17389fCb843a;
     address public constant MAINNET_SYMBIOTIC_SUSDE_COLLATERAL = 0x19d0D8e6294B7a04a2733FE433444704B791939A;
+    address public constant MAINNET_SYMBIOTIC_MEV_CAPITAL_WSTETH_COLLATERAL = 0x4e0554959A631B3D3938ffC158e0a7b2124aF9c5;
```

Comment:
Similar to [https://github.com/sense-finance/rumpel-wallet/pull/41/files#r1953753194](https://github.com/sense-finance/rumpel-wallet/pull/41/files#r1953753194), this contract doesn't seem to have a transfer function. are you seeing the same?

Reply:
yes - will review and update

Reply:
updated